.github/workflows/ci.yml

```
@@ -113,9 +131,10 @@ jobs:
   playwright-test-failure-comment:
     name: Share Playwright test debugging instructions
     runs-on: ubuntu-latest
-    if: always() && github.ref != 'refs/heads/main' && (needs.nuxt-playwright.result == 'failure' || needs.storybook-playwright.result == 'failure')
+    if: always() && github.ref != 'refs/heads/main' && (needs.nuxt-playwright-vr.result == 'failure' || needs.nuxt-playwright-e2e.result == 'failure' ||needs.storybook-playwright.result == 'failure')
     needs:
-      - nuxt-playwright
+      - nuxt-playwright-e2e
+      - nuxt-playwright-vr
       - storybook-playwright
```

In the interest of clarity
```suggestion
      - storybook-playwright-vr
```